tests/CMakeLists.txt

```diff
@@ -294,11 +294,13 @@ if(ENABLE_MOCK_SERVER_TESTS)
     add_net_test_case(request_time_too_skewed_mock_server)
 endif()
 
-add_net_test_case(s3express_provider_long_run_real_server)
-add_net_test_case(s3express_client_put_test_small_real_server)
-add_net_test_case(s3express_client_put_test_large_real_server)
-add_net_test_case(s3express_client_put_long_running_test_real_server)
-add_net_test_case(s3express_client_get_test_real_server)
+add_net_test_case(s3express_provider_long_running)
+add_net_test_case(s3express_client_put_test_small)
+add_net_test_case(s3express_client_put_test_large)
+add_net_test_case(s3express_client_put_test_multiple)
+add_net_test_case(s3express_client_put_long_running_test)
+add_net_test_case(s3express_client_get_test)
+add_net_test_case(s3express_client_get_test_multiple)
```

Comment:
trivial / debatable: I prefer verbosity in test names, such as mentioning MPU/NON-MPU, the number of requests in multiples, and what is meant by a long-running test.